### DISCUSSION TITLE
WinDriver.sys и почему при запуске анти-вирус индефинцирует программу как троян?

### DISCUSSION THREAD

**Discussion by venysuperr** [bol-van/zapret#845] [Q&A]
WinDriver.sys при его запуске программу больше нельзя удалить и анти-вирус говорит что в папке присутствует роян и в названии WinDriver.sys появляется приписка Bitcoin и ссылка

**Comment by bol-van** [comment]
Антивирусы могут жаловаться на windivert, но это ложная тревога.
Если прога была скачана не отсюда, то могут быть трояны и майнеры. Гарантий нет
